StudentDetails.java

```diff
@@ -0,0 +1,155 @@
+import java.util.*;
+import java.util.Scanner;
+
+class StudentData {
+    private int sno;
+    private String sfname;
+    private String slname;
+
+    StudentData(int sno, String sfname, String slname) {
+        this.sno = sno;
+        this.sfname = sfname;
+        this.slname = slname;
+    }
+
+    public int getSno() {
+        return sno;
+    }
+
+    public String getSlname() {
+        return slname;
+    }
+
+    public String getSfname() {
+        return sfname;
+    }
+
+    public String toString() {
+        return sno + " " + sfname + " " + slname;
+    }
+}
+
+class StudentDetails {
```

Comment:
Without knowing the code what is the difference between __StudentData__ and __StudentDetails__? You cannot tell, since the name does not add any clue or hint. Since __StudentDetails__ is just the main entry point to your application, go and use the Java convention and use the name __Main__ for this class instead of __StudentDetails__. Also why use of default access modifier? And the third thing is __place each class file in it's own .java file__.